jdbc/src/main/java/io/kestra/jdbc/repository/AbstractJdbcTriggerRepository.java

```diff
@@ -287,24 +287,41 @@ public Trigger lock(String triggerUid, Function<Trigger, Trigger> function) {
                 return null;
             });
     }
+
     @Override
     public ArrayListTotal<Trigger> find(Pageable pageable, String tenantId, List<QueryFilter> filters) {
         return this.jdbcRepository
             .getDslContextWrapper()
             .transactionResult(configuration -> {
                 DSLContext context = DSL.using(configuration);
-                SelectConditionStep<?> select = generateSelect(context, tenantId, filters);
+                SelectConditionStep<org.jooq.Record> select = generateSelect(context, tenantId, filters);
```

Comment:
```suggestion
                SelectConditionStep<Record> select = generateSelect(context, tenantId, filters);
```

Please avoid packages in the code, it make it harder to read